rest-service/manager_rest/resource_manager.py

```diff
@@ -593,6 +761,10 @@ def cancel_execution(self, execution_id, force=False, kill=False):
         if kill:
             force = True
         execution = self.sm.get(models.Execution, execution_id)
+        # When a user cancels queued execution automatically use the kill flag
+        if execution.status == ExecutionState.QUEUED:
+            kill = True
```

Comment:
hmm, tbh shouldn't be a reason to kill, does it actually change anything for queued executions?

Reply:
It won't be executed if it's status isn't `queued`. So if a user wants to cancel a queued execution he will use `cfy exec cancel` with no flags (and we will add them in the rest).